Eatery Blue/Assets.xcassets/Calendar 1.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Menu-button (1).png",
```

Comment:
Can you fix this?

Reply:
lowkey I have no idea where that is cant find it in the codebase maybe deleted unknowingly